Gems/AtomLyIntegration/CommonFeatures/Code/Source/PostProcess/ChromaticAberration/ChromaticAberrationComponentController.cpp

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <PostProcess/ChromaticAberration/ChromaticAberrationComponentController.h>
+
+#include <AzCore/RTTI/BehaviorContext.h>
+
+#include <Atom/RPI.Public/Scene.h>
+
+namespace AZ
+{
+    namespace Render
+    {
+        void ChromaticAberrationComponentController::Reflect(ReflectContext* context)
+        {
+            ChromaticAberrationComponentConfig::Reflect(context);
+
+            if (auto* serializeContext = azrtti_cast<SerializeContext*>(context))
+            {
+                serializeContext->Class<ChromaticAberrationComponentController>()->Version(0)->Field(
+                    "Configuration", &ChromaticAberrationComponentController::m_configuration);
```

Comment:
```suggestion
                serializeContext->Class<ChromaticAberrationComponentController>()
                    ->Version(0)
                    ->Field( "Configuration", &ChromaticAberrationComponentController::m_configuration);
```

Normally, the version and fields are placed on separate lines.

Reply:
Well spotted! Fixed in the commit I am about to push.